shell/platform/windows/cursor_handler.cc

```diff
@@ -11,8 +11,14 @@
 static constexpr char kChannelName[] = "flutter/mousecursor";
 
 static constexpr char kActivateSystemCursorMethod[] = "activateSystemCursor";
+static constexpr char kSetSystemCursorMethod[] = "setSystemCursor";
 
 static constexpr char kKindKey[] = "kind";
+static constexpr char kCustomCursorBufferKey[] = "buffer";
+static constexpr char kCustomCursorHotxKey[] = "hotx";
+static constexpr char kCustomCursorHotyKey[] = "hoty";
+static constexpr char kCustomCursorWidthKey[] = "width";
+static constexpr char kCustomCursorHeightKey[] = "height";
```

Comment:
```suggestion
static constexpr char kActivateSystemCursorMethod[] = "activateSystemCursor";
static constexpr char kKindKey[] = "kind";

// Also: Can you document this method here (the arguments and their types, and return format.)
static constexpr char kSetSystemCursorMethod[] = "setSystemCursor";
static constexpr char kCustomCursorBufferKey[] = "buffer";
static constexpr char kCustomCursorHotxKey[] = "hotx";
static constexpr char kCustomCursorHotyKey[] = "hoty";
static constexpr char kCustomCursorWidthKey[] = "width";
static constexpr char kCustomCursorHeightKey[] = "height";
```

Reply:
Also why is this new method called `set system cursor` while it's actually a custom cursor?

Reply:
Maybe "setCustomCursor" is more suitable for this method?